app/models/provider.rb

```diff
@@ -11,7 +11,7 @@
 #  provider_code        :text
 #  provider_type        :text
 #  postcode             :text
-#  url                  :text
+#  website              :text
```

Comment:
Where's the rest of the stuff added? `train_with_us` etc

Reply:
good question.... an even better one is what happened to `db:annotate`

![image](https://user-images.githubusercontent.com/470137/67214028-f5189e80-f416-11e9-8fbd-de08e310b7fe.png)


manage-courses-backend$ rake -T | grep "rake db:"
manage-courses-backend$ bundle exec rake -T | grep "rake db:"